config.yaml.defaults

```diff
@@ -119,6 +121,10 @@ app:
       icon: LocalCafe
       url: /runs
 
+    - name: Allocations
+      icon: Satellite
```

Comment:
Maybe some clock icon? there must be a bunch of time-related icons that better fit this.

Reply:
made it an hourglass.